schema/cassandra/cadence/versioned/v0.35/partition_config.cql

```diff
@@ -1,9 +1,5 @@
 ALTER TYPE workflow_execution ADD partition_config map<text, text>;
 ALTER TYPE task ADD partition_config map<text, text>;
 
-CREATE TYPE isolation_group_partition (
-  name text,
-  state int,
-);
-
-ALTER TYPE domain_config ADD isolation_groups frozen<list<isolation_group_partition>>;
+ALTER TYPE domain_config ADD isolation_groups blob;
```

Comment:
could you make the change to v0.36?
I have updated `staging2` to v0.35, and it's hard to apply the new changes with the schema upgrade tool without updating the version?

Reply:
yes, completely reasonable